arex-instrumentation-foundation/src/main/java/io/arex/foundation/config/ConfigManager.java

```diff
@@ -500,6 +503,26 @@ private long nextWorkTime() {
         return Duration.between(LocalDateTime.now(), nextTime).toMillis();
     }
 
+
+    public List<String> getIgnoredClassLoaders() {
+        return ignoredClassLoaders;
+    }
+
+    public void setIgnoredClassLoaders(String ignoredClassLoaders) {
+        if (StringUtil.isEmpty(ignoredClassLoaders)) {
+            if (this.ignoredClassLoaders == null) {
+                this.ignoredClassLoaders = Collections.singletonList(ArexConstants.OT_AGENT_CLASSLOADER);
+            }
+            return;
+        }
```

Comment:
ignoredClassLoaders is null, set ignoredClassLoaders is `Collections.emptyList()`